### PR TITLE
Adding LabTech

### PR DESCRIPTION
**Pull request by guygregory**

### CODE REVIEW

**Comment by mxxcon** [comment]
From the documentation link, it seems like this is a self-hosted software product. Am I correct? Generally this site does not list such.

**Comment by guygregory** [comment]
It's offered as either self-hosted (LabTech) or SaaS (LabTech Cloud). The documentation is the same in both cases.

**Comment by guygregory** [comment]
Either way, the actual software used (LabTech Control Center) is installed on a local PC. If you need any more info to help you decide suitability, please let me know.

File: _data/remote.yml [hunk 1]
```diff
@@ -15,6 +15,14 @@ websites:
       software: Yes
       doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_common_Security_TwoStep.html
 
+    - name: LabTech Control Center
+      url: http://www.labtechsoftware.com/
```
Comment (Carlgo11) [inline _data/remote.yml]: Seems like they support https. Please change the URL to `https://www.labtechsoftware.com/` :wink:

File: _data/remote.yml [hunk 2]
```diff
@@ -15,6 +15,14 @@ websites:
...
+      img: labtech.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
```
Comment (Carlgo11) [inline _data/remote.yml:23]: I can't find anything relating to hardware based authentication on the doc page or on AuthAnvil's official page.
Maybe you could point me in the right direction?

Comment (guygregory) [inline _data/remote.yml:23]: https://www.scorpionsoft.com/tokens

**Comment by RichJeanes** [comment]
Is there any documentation for end users to enable 2FA on their accounts and/or a page that actually lists the available types of 2FA? 

The currently linked documentation says

> Your employees will input their chosen PIN and then a one-time is generated from an AuthAnvil app on their smartphone

Is there any documentation of hardware 2FA support?

**Pull request closed without merging**